brooklyn-tosca-transformer/src/test/java/io/cloudsoft/tosca/a4c/Alien4CloudIntegrationTest.java

```diff
@@ -40,10 +43,7 @@ public void setUp() throws Exception {
                 .useManagementContext(mgmt)
                 .launch()
                 .getCampPlatform();
-        Alien4CloudToscaPlatform.grantAdminAuth();
 
-        this.platform = super.applicationContext.getBean(Alien4CloudToscaPlatform.class);
-        platform.loadNormativeTypes();
```

Comment:
[this](https://github.com/cloudsoft/brooklyn-tosca/blob/master/brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/ToscaPlanToSpecTransformer.java#L101) null check is false, therefore actually uses the platform that is intended